compiler/luci/profile/src/CircleNodeOrigin.cpp

```diff
@@ -144,13 +144,18 @@ bool has_origin(const luci::CircleNode *circle_node)
   return circle_node->annot<CircleNodeOriginAnnotation>() != nullptr;
 }
 
+/**
+ * @brief 'origin' is added to the existing origin of circle_node.
+ * @note  If 'origin' is nullptr, nothing changed because there is nothing to be added.
```

Comment:
Are you going to leave this like this? If so, I sugguest not to. It would be better to fix caller (maybe later) not to call this methof if there is nothing to add (nullptr).
If not, please change this comment and add a TODO for `if (origin == nullptr)` line.

Reply:
Now `origin` can be `nullptr` and it does not cause errors because we changed the logic of `add_origin`.
I missed this point, thanks :)

I will remove the comment and `nullptr` check...!